src/moodbar/moodbarbuilder.cpp

```diff
@@ -187,9 +187,9 @@ QByteArray MoodbarBuilder::Finish(int width) {
 
     const int n = end - start;
 
-    *(data++) = rgb.r / n;
-    *(data++) = rgb.g / n;
-    *(data++) = rgb.b / n;
+    *(data++) = char(rgb.r / n);
```

Comment:
Please test this.
I did this previously, and the moodbar didn't work afterwards.
I reverted the change as part of https://github.com/strawberrymusicplayer/strawberry/commit/9cde537066c6332495e94c6bd2b4b7bc48ab32b8#diff-a20292def67aa3d74f22d1010c269c5420c5e0f93b758dea630c8b55eb4a2fff

Reply:
As far as I can see the change works here. So it is probably good.